### PR TITLE
Update README to remove legacy terra-toolkit documentation

### LINKED ISSUE #404: Links for installing terra-toolkit needs update
# Issue Description
The documentation and git header link is pointing to [terra-toolkit-boneyard](https://github.com/cerner/terra-toolkit-boneyard) instead of terra-toolkit

### PR DESCRIPTION
**Pull request by StephenEsser**
### Summary

This PR updates the README to remove the legacy terra-toolkit documentation. 

Resolves #404

## Additional Details

The rendered markdown can be demoed here: https://github.com/cerner/terra-toolkit/blob/readme-update/README.md

@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

**Review by emilyrohrbough — APPROVED** [review]
Totally random thought, if teams wanted to consumer terra-toolkit directly, could we release it as a package that uses all of the packages defined in this repo? Is that possible?

**Comment by StephenEsser** [comment]
An owner of the repository will need to update the repository heading and homepage as part of this change. (The description might be fine left alone, but the homepage may confuse users)

image

**Comment by mjhenkes** [comment]
@emilyrohrbough, possible, yes, but it kind of defeats the purpose of breaking it up.

**Review by mjhenkes — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -8,64 +8,39 @@
...
-
-See the [Serve Guide](https://github.com/cerner/terra-toolkit/blob/master/scripts/serve/Serve.md) to get started.
-
-## Webdriver.io Utility
-
-[Webdriver.io](http://v4.webdriver.io/) is a framework for writing webdriver powered tests to validate functionality in browsers. The Webdriver.io framework provides services for setting up a selenium server, starting webpack and static servers, running accessibility and visual regression testing, and more.
+## Notice
 
-See the [Webdriver.io Utility Developer's Guide](https://github.com/cerner/terra-toolkit/blob/master/docs/Wdio_Utility.md) to get started.
+The code for the [terra-toolkit](https://www.npmjs.com/package/terra-toolkit) npm dependency has been moved to [terra-toolkit-boneyard](https://github.com/cerner/terra-toolkit-boneyard).
```
Comment (yuderekyu) [inline README.md:27]: can we note that this code is currently in development?

```suggestion
This code is currently in development. The code for the [terra-toolkit](https://www.npmjs.com/package/terra-toolkit) npm dependency has been moved to [terra-toolkit-boneyard](https://github.com/cerner/terra-toolkit-boneyard).
```

**Review by yuderekyu — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (StephenEsser) [reply README.md:27]: What is `this` in this context? Are you wanting to add a note that terra-toolkit package code is still being developed in terra-toolkit-boneyard? Or that terra-toolkit in this repository is under development?

Comment (yuderekyu) [reply README.md:27]: the terra-toolkit in this repo

Comment (StephenEsser) [reply README.md:27]: Can you provide a few more details about adding the development notes? There are two packages in this repo that are released artifacts on npm. Generally all repository code falls under "in development". Are you wanting to clarify that the terra-functional-testing package specifically is under development and not yet released?

Comment (yuderekyu) [reply README.md:27]: ah, yes, unreleased is a better term. not a huge deal tho

Comment (StephenEsser) [reply README.md:27]: I think I'll leave it as is for now. The browserslist-config-terra and eslint-config-terra are stable releases from this repository. Only the terra-functional-testing package is unreleased right now so the note doesn't apply to everything. We can circle back around if we want to update the README with additional information.  For now I'd like to get this merged to help guide people to terra-toolkit-boneyard if that is what they are looking for.

**Pull request merged**